mkdocs_lumi/main.html

```diff
@@ -72,6 +72,23 @@
       href="https://use.typekit.net/nlo5lta.css?ver=5.5.3"
       type="text/css" media="all"
     />
+
+  <script>
+    const modifyExternalLinksToOpenInNewTab = () => {
+      const anchorElements = Array.from(document.getElementsByTagName("a"))
+
+      const anchorElementsWithHrefPointingOutside = anchorElements
+        .filter(a => a.host && a.host !== window.location.host)
```

Comment:
To avoid overwriting any existing, intentionally defined values of the _target_ attribute, one could add another conjunct to test whether the attribute in question already exists:
```suggestion
        .filter(a => a.host && a.host !== window.location.host && !a.hasAttribute("target"))
```